### DISCUSSION TITLE
Help me understand possible benefits moving to frigate

### DISCUSSION THREAD

**Discussion by Nuendo-DE** [blakeblackshear/frigate#9240] [Ask A Question] [answered]
Hi there,

I'm currently using some Hikvision cameras with QNAP's QVR Pro application. My cameras support smart events like line crossing detection, intrusion detection and some more.

These are examples of cameras I'm using:

https://www.hikvision.com/de/products/IP-Products/Network-Cameras/Pro-Series-EasyIP-/ds-2cd2t86g2-2i-4i/

In case of an event detected by the camera the recording in QVR Pro is triggered. So there's no intelligence or cpu resources needed on the NAS as the whole detection is done on the cameras itself.

Would someone be so kind and help me understand if moving to frigate could make sense nonetheless? I was able to get a Coral M2 expansion for a decent price for my NAS so technically I would be able to give frigate a try.

Thank you!

**Comment by NickM-27** [comment, marked as answer]
Moving to frigate making sense or not depends on you. If you are totally satisfied with the way your system works, why change? On the other hand, if further control of detection behavior, integration with other platforms like MQTT (and NodeRED / HomeAssistant by extension), or many other features that frigate offers are enticing then that would be the reason to give frigate a try.

**Reply by NickM-27** [reply]
There is a demo site that you can use to play around with frigate https://demo.frigate.video
